directives/editInline.js

```diff
@@ -229,7 +229,13 @@
                   switch (event.type) {
                     case 'focusout':
                     case 'blur':
-                      $scope.$apply($scope.accept)
+                      /// below code will attached stratus route main if user try to create new article route
+                        if($scope.model.target === 'Article'){
```

Comment:
This is really specific and shouldn't be in the common directives.  If you can move this into your controller or whatever have you for specificity, that would be much better.

Reply:
I tried to watch this in selectMainRoute but its not propagating..  The problem is that .. since this is part of Article..its not attached any controller/model.. I have tried to come-up with best solution but failed.. so i thought of handling blur here is better incase of Article route..